examples/TicTacToe.bgl

```diff
@@ -8,6 +8,8 @@ type Position = (Int,Int)
 type Board = Array (3,3) of {X, O, Empty}
 type Input = Position
 
+type Result = Player & {Tie}
```

Comment:
As a sidenote, I've already started doing this in any bogl examples I write up for the past month or so. It's also helpful for new students who may not immediately realize what 'Player & {Tie}' represents.